lib/src/services.dart

```diff
@@ -34,39 +35,43 @@ class Services extends InheritedWidget {
   bool updateShouldNotify(InheritedWidget oldWidget) {
     return false;
   }
-///
+
+  ///
   static Services of(BuildContext context) {
-    return context.dependOnInheritedWidgetOfExactType<Services>()!;
+    return context.dependOnInheritedWidgetOfExactType<Services>()
+        as Services; // WHAAAT?
```

Comment:
You do not need the Services InheritedWidget anymore.

Once you pick a state management system -- stick to it. A mix of different approaches is hard to read and maintain.